db/schema.rb

```diff
@@ -11,11 +11,31 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150414190858) do
+ActiveRecord::Schema.define(version: 20150415195410) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "actions", force: :cascade do |t|
+    t.integer  "person_id",   null: false
+    t.integer  "activity_id", null: false
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.